stdlib/public/Python/PythonLibrary.swift

```diff
@@ -0,0 +1,200 @@
+//===-- PythonLibrary.swift -----------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements the logic for dynamically loading Python at runtime.
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Darwin)
+import Darwin
+#elseif canImport(Glibc)
+import Glibc
+#endif
+
+//===----------------------------------------------------------------------===//
+// The `PythonLibrary` struct that loads Python symbols at runtime.
+//===----------------------------------------------------------------------===//
+
+struct PythonLibrary {
+  
+  private static let shared = PythonLibrary()
+  private static let pythonLegacySymbolName = "PyString_AsString"
+  
+  private let pythonLibraryHandle: UnsafeMutableRawPointer
+  private let isLegacyPython: Bool
+  
+  fileprivate init() {
+    guard let pythonLibraryHandle = PythonLibrary.getPythonLibraryHandle() else {
+      fatalError(
+        "Python library not found. Set the \(Environment.library.key) " +
```

Comment:
Please use a multi-line string literal instead of concatenation.

Reply:
You are looking for this functionality:
```swift
fatalError("""
  Python library not found. Set the \(Environment.library.key) \
  environment variable with the path to the Python Library.
  """)
```
Note the `\` at the end of the first line.